Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1009,11 +1009,16 @@ void MenuBar::UpdateToolsMenu(bool emulation_started)
     IOS::HLE::Kernel ios;
     const auto tmd = ios.GetES()->FindInstalledTMD(Titles::SYSTEM_MENU);
 
-    const QString sysmenu_version =
-        tmd.IsValid() ?
-            QString::fromStdString(DiscIO::GetSysMenuVersionString(tmd.GetTitleVersion())) :
-            QString{};
-    m_boot_sysmenu->setText(tr("Load Wii System Menu %1").arg(sysmenu_version));
+    std::string sysmenu_version;
+    if (tmd.IsValid())
+    {
+      sysmenu_version = DiscIO::GetSysMenuVersionString(tmd.GetTitleVersion());
+
+      if (tmd.IsvWii())
+        sysmenu_version += " (vWii)";
```

Comment:
To enable more flexibility for translators:

```suggestion
        sysmenu_version = tr("%1 (vWii)");
```

(Also do the same for Android, using `Common::FmtFormatT` instead of `tr`)

Reply:
Hmm, thoughts about making 2 separate translations for `Load Wii System Menu %1` and `Load vWii System Menu %1`?
That way the `(vWii)` suffix could be dropped entirely.

Reply:
That would also be fine.